crates/types/src/adapters/models.rs

```diff
@@ -236,19 +237,31 @@ pub struct GetOrderResponse {
 	pub order: OrderResponse,
 }
 
+/// Response containing order details.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[cfg_attr(feature = "openapi", derive(ToSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct SubmitOrderResponse {
```

Comment:
If create order endpoint starts to return full order body this type could be deleted